src/tokens/d4b29d5d48edf300128a5f08150ad196a6af15ebd7932306d3099efe424f4f.yaml

```
@@ -0,0 +1,35 @@
+verified: true # default true, if a token violate verification policy then switch to false
+
+# the following fields are not required
+# for `number`, it's token number with no decimals. For example, if your token has a max supply of 50,000,000 tokens with 6 decimals, the value needs to be 50000000 × 10^6 = 50000000000000
+# for URL, it must also return the token number without decimals
+maxSupply: 1000000000
+
+treasury:
+  - addr1xyq20a3z0ef3xpps0r4tny54v0z03e5lgwaezqgx6gfqamt3phpm0dmrcz4he7fuglvhdx4nhnks22u42arkugw2svnq0ywtnv
+
+burn:
+  - addr1w8qmxkacjdffxah0l3qg8hq2pmvs58q8lcy42zy9kda2ylc6dy5r4
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)